README.md

```diff
@@ -1 +1,64 @@
-# book-store
+# Book store <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\logo small.png"/>
+
+The project was established with the goal to help book lovers to choose and to buy books online. 
+Inspired by Amazon of the early days. Book-store service is written in Java and this means that it has unlimited
+potential for further development. By now it has only basic functionality. 
+The selection of books is very large, but the interface is easy to understand and not overloaded with functionality,
+so you can quickly find the book you like.
+
+# Technologies been used <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\spring logo.png"/> <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\java 17 logo.png"/>
```

Comment:
Recheck your readme on any kind of issues. 

Here is an example of grammar issue. 
```suggestion
# Technologies have been used
```